lib/index.js

```diff
@@ -58,6 +61,16 @@ export const mapComponentState = {
     })
 };
 
+export const middleware = store => next => action => {
+  const { dispatch, getState } = store;
```

Comment:
Is this being linted? I don't see `getState` being used

Reply:
Ah, good catch. Will fix.